presto-main/src/test/java/com/facebook/presto/spiller/TestFileSingleStreamSpiller.java

```diff
@@ -78,6 +101,14 @@ public void testSpill()
         spiller.spill(Iterators.forArray(page, page, page)).get();
         assertEquals(listFiles(spillPath.toPath()).size(), 1);
 
+        // Assert the spill codec flags match the expected configuration
+        try (InputStream is = java.nio.file.Files.newInputStream(listFiles(spillPath.toPath()).get(0))) {
+            Iterator<SerializedPage> serializedPages = PagesSerdeUtil.readSerializedPages(new InputStreamSliceInput(is));
+            assertTrue(serializedPages.hasNext(), "at least one page should be successfully read back");
+            SerializedPage readBack = serializedPages.next();
```

Comment:
readBack sounds strange; reconstructedPage might be better; better yet, simplify to

```
            byte marker = serializedPages.next().getPageCodecMarkers();
            assertEquals(PageCodecMarker.COMPRESSED.isSet(marker), compression);
            assertEquals(PageCodecMarker.ENCRYPTED.isSet(marker), encryption);
```

Reply:
Looks better, will use this instead.